### ISSUE TITLE
Add support for showing bitrate

### ISSUE THREAD

**Issue by denbeigh2000** [tchaik/tchaik#35]
It would be cool to show bitrates and (if possible) encodings in the Web UI.

If possible, we should differentiate between fixed and variable-rate compression

**Comment by solarnz** [comment]
Is this for showing the bitrate in the catalogue, or is it for showing the bitrate while you're playing the file?

**Comment by denbeigh2000** [comment]
Ideally in the catalogue.

**Comment by dhowden** [comment]
We need to now implement something in the frontend to display it.

The iTunes importer will pick this value up if set by iTunes, but we now need a good way to extract this from audio files which are imported using the path walker.

**Comment by solarnz** [comment]
How do we plan on supporting variable bitrates? Do we want to show MP3 V0 rather than an actual bitrate value?

Maybe showing that it is V0 should be part of how we should what format the file is in.

**Comment by dhowden** [comment]
Yes, so having variable bit rates is an issue - and it would seem that in this case iTunes shows the average.  Apparently my commit message wasn't clear enough and GitHub chose to close this issue :-).

In future the Tchaik library should probably have a better type for the "BitRate" field, it would seem that an integer isn't enough here.

For the moment, it would seem that the next step would be to implement reading this from the underlying audio file (hopefully MP3, MP4/ALAC and FLAC) and then once that is done change the library format for the BitRate field.

**Comment by xhenner** [comment]
I'm adding support in dhowden/tag for calculating bitrate for the different formats (I need it for my own project). 
I had a different library to do the job, but It make sense to add the support into tag directly
